### DISCUSSION TITLE
[Feature] Make people thumbnail crop zoom configurable

### DISCUSSION THREAD

**Discussion by edwardgalligan** [immich-app/immich#11982] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

This may be an aesthetic preference but to me the default zoom out for cropping face-detected thumbs of people in images for the "Featured photo" is far too tight: the resulting thumbs are too zoomed in & look terrible due to lack of spacing around faces.

I see [that the zoom out figure is set to 10% and hardcoded](https://github.com/immich-app/immich/blob/main/server/src/services/person.service.ts#L684-L685) - it would be great if this value were configurable

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by ireun** [comment]
This is not purely aesthetic - it also makes manual merging difficult. So yeah! Thumbs up from me!
